packages/qdrant-loader/src/qdrant_loader/core/worker/pool.py

```
@@ -0,0 +1,205 @@
+                async with source_lock:
+                    logger.info(
+                        "job.handler_started",
+                        job_id=job.id,
+                        job_type=job.type,
+                        source_key=source_key,
+                        attempt=job.attempts,
+                    )
+                    t0 = time.monotonic()
+                    try:
+                        await self._handler(job.type, payload)
+                    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)